### PR TITLE
billie: config.fs: Checkout changes from CLO

### PR DESCRIPTION
**Pull request by hridaya2004**
- Checked out changes from tag : LA.UM.9.12.r1-14700.03-SMxx50.QSSI12.0

Signed-off-by: kawaaii <kawaaii@nocturn9x.space>
Change-Id: I21bd5e07f6d1125081db57d8a23198b620d27eda

### CODE REVIEW

**Review by danascape — CHANGES_REQUESTED** [review]
'system/vendor/' is symlinked with 'vendor/' so there is no need to keep both of the mappings. Kindly clean then up.

**Review by saalim-nimo — APPROVED** [review]
Drop the wcnss_filter entry, the binary does not exist.

**Review by danascape — CHANGES_REQUESTED** [review]
wcnss_filter mapping is unneeded

File: config.fs [hunk 1]
```diff
@@ -19,22 +19,40 @@ value:2906
...
-[vendor/bin/ims_rtp_daemon]
+[vendor/bin/hw/android.hardware.bluetooth@1.0-service-qti-lazy]
 mode: 0755
-user: AID_RADIO
-group: AID_RADIO
+user: AID_BLUETOOTH
+group: AID_BLUETOOTH
+caps: BLOCK_SUSPEND NET_ADMIN
+
+[system/bin/cnss-daemon]
```
Comment (danascape) [inline config.fs]: This doesnt exist in system

**Review by danascape — APPROVED** [review]

**Pull request merged**